### ISSUE TITLE
Creation of a Place to Write the First Series 4 Paper

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMalaria/OSM_To_Do_List#288]
@miike can you please give me a quick hand? Trying to set up a repo where we can share files (data, Chemdraw files, png images) and Word documents related to the writing of the first paper on Series 4. Seems like we ought to be able to use GH as a super-Dropbox. I created a repo

https://github.com/OpenSourceMalaria/OSMSeries4Paper1

But it feels like I am not doing something that I need to do:

![screen shot 2015-03-27 at 9 35 14 am](https://cloud.githubusercontent.com/assets/4386101/6858637/c2a2563e-d464-11e4-8f62-fe8e4be3411b.png)

What's the next step before we can start adding files?

**Comment by miike** [comment]
In order to do that you'll want the 'Setup in Desktop' button which should set the repo up automatically in the Github Mac/Windows client. It's not quite as simple as Dropbox as after each change changes will need to be pushed ("synced") in the client to the repository.

For the client it will ask you for a local directory and then any changes to that directory will show up under 'Uncommitted changes'. Once that has been done clicking 'Commit to master' (it's probably safe enough for the moment if everyone commits to the same master (main) branch) will commit those changes locally. Finally clicking 'Publish' in the top right of the client will push those commits to the repository on Github.

**Comment by mattodd** [comment]
So the short story is that I need the desktop client. The files are hosted there, and are mirrored on the web version (which people see) when I (as repo owner) "push" them. Anyone can change the files and "commit to master" which will change only the files on my computer. Is that the gist?

**Comment by miike** [comment]
Sort of, but the copy on Github (in this case the OSMSeries4Paper1 which is owned by OpenSourceMalaria) is the source of truth (often referred to as the *upstream* remote). When you create a mirror *clone* of this is a local copy which you can make changes to. When you make these changes you can either write directly to the repository by *pushing* your changes (shared repository model) or you can *fork* (create your own copy that isn't owned by OSM) the repository. When you fork the repository it is into your own personal account and you gain read and write access. You can make changes to your local copy and then push (write) to your personal remote on Github (often referred to as the *origin* remote). That may not be 100% clear but Github has a nicer explanation [here](https://help.github.com/articles/using-pull-requests/) under 'Types of collaborative development models'.

The shared repo model means that many people need write access to the OSM repositories which for large projects doesn't tend to work well. The fork & pull model means that users do not need write access to the OSM repo (only the owners have write access) and requested changes come through a *pull request* which is moderated by users with write access.

**Comment by egonw** [comment]
Miike, on GitHub you can make teams and have only write access to one specific repository for that team. So, make a series4-paper-writers team?

**Comment by miike** [comment]
That's definitely a possibility that would simplify things a fair bit.

**Comment by mattodd** [comment]
OK, thanks Mike and Egon. Github installed and I've managed to push the current paper files to the repo

https://github.com/OpenSourceMalaria/OSMSeries4Paper1

I'll check I can create a writing "team" who will have control of the files, however anyone should be able to see and download any of the files, right? Meaning that any changes made the old-fashioned way can just be mailed through as per usual. Closing this for now - thank you for your help with this. It's going to be an interesting experiment in paper writing.
